packages/vaex-core/vaex/expression.py

```diff
@@ -276,6 +276,61 @@ def max(self, binby=[], limits=None, shape=default_shape, selection=False, delay
         kwargs['expression'] = self.expression
         return self.ds.max(**kwargs)
 
+    def value_counts(self, dropna=False, ascending=False):
+        """Computes counts of unique values
+         WARNING:
+          * If the expression/column is not categorical, it will be converted on the fly
+          * dropna is False by default, it is True by default in pandas
+         :param dropna: when True, it will not report the missing values
+        :param ascending: when False (default) it will report the most frequent occuring item first
+        :returns: Pandas series containing the counts
+        """
+        from pandas import Series
+        df = self.ds
+        # whatever we do, it's gonna be similar to ds.categorise, so better
+        # to use that path
+        if not df.iscategory(self.expression):
+            df = df.copy()
```

Comment:
I don't understand what is the purpose of this line.. can you please explain?

Reply:
I thought inplace=True was default, which is not